hawkular-command-gateway-war/pom.xml

```diff
@@ -72,11 +72,6 @@
     </dependency>
 
     <dependency>
-      <groupId>org.hawkular.inventory</groupId>
-      <artifactId>hawkular-inventory-json-helper</artifactId>
-    </dependency>
```

Comment:
This is a side effect of having a `recipientType` and `recipient`: if `recipientType` is `FEED`, then `recipient` is supposed to be a feedId. The question is how practical will it be for clients to need to know the feedIds. It tend to think that clients should be able to know the feedId they want to speak to. WDYT?